charts/sda-svc/templates/_helpers.yaml

```diff
@@ -198,6 +198,12 @@ Create chart name and version as used by the chart label.
 {{- end -}}
 
 {{/**/}}
+{{- define "dbUserInbox" -}}
+{{- ternary "lega_in" .Values.credentials.inbox.dbUser (empty .Values.credentials.inbox.dbUser) -}}
```

Comment:
Not wrong, but `default` looks more obvious to me.

Reply:
Maybe we can update that everywhere in a separate commit?

Reply:
Fine with me.